.github/workflows/main.yml

```diff
@@ -29,6 +29,12 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Setup Working Directory
+      run: |
+        echo "Current Working Directory is $(pwd)"
+        echo "Adding PWD to Python path"
+        echo "env:PYTHONPATH=$env:PYTHONPATH;$(pwd)" | Out-File -Append -FilePath $env:GITHUB_ENV -Encoding utf8
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
```

Comment:
below you could put
```
          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
          if [ -f requirements_dev.txt ]; then pip install -r requirements_dev.txt; fi
```

to install requirements.txt and requirements_dev.txt, and, remove below `pip install pylint flake8 mypy pytest`

Reply:
Good idea